modules/custom/reservation/templates/reservation.html.twig

```diff
@@ -35,16 +33,25 @@
                         <center><h2>Confirm your resrvation</h2><center>
                     </div>
                     <div class="modal-body">
-                        <h4 id="movie-title-{{movie.nid.value}}" style = "text-align:center;"> {{movie.title.value | raw}}  </h4>
-                        <center><h5>Select day:</h5></center>
-                        <center><form>
-                        {% for day in movie.field_available_days %}
-                            {% set seats = movie.field_attendants_ %}
-                            <input type="radio" name="day" value="{{seats.value}}" class="popupInput">
-                            <label>{{day.entity.name.value | raw}} - Available seats : {{seats.value}}</label><br>
-                        {% endfor %}
-                        <center><button name="button" type="button" class="confirmButton" >Confirm your movie reservation</button></center>
-                        </form></center>
+                        <center>
+                        <form id="popUpForm" class="popUpForm" method="post">
+                            <h4 id="movie-title-{{movie.nid.value}}" style = "text-align:center;"> {{movie.title.value | raw}}  </h4>
+                            <center><h5>Select day:</h5></center>
+                            {% for day in movie.field_available_days %}
+                                {% set seats = movie.field_attendants_ %}
+                                <input  type="radio" name="day" value="{{day.entity.name.value | raw}}" class="popupInput">
+                                <label>{{day.entity.name.value | raw}} - Available seats : {{seats.value}}</label><br>
+                            {% endfor %}
+
+                            <input type="hidden" class="movieTitle"  name="title" value="{{movie.title.value | raw}}" >
+                            <input class="hiddenName" id="hiddenName" type="hidden" name="name" required >
```

Comment:
Da li ima smisla da imamo hidden field koji je skriven? Kako korisnik da zna za to?